src/01-functions.js

```diff
@@ -168,7 +195,12 @@ function joinStringsWithSpaces() {}
  *  getFirstAndLastCharacter("upwards");
  *  //> "us"
  */
-function getFirstAndLastCharacter() {}
+function getFirstAndLastCharacter(a) {
+  let First = a.charAt(0);
+  let Second = a.charAt(a.length -1);
+  let Boffum = `${First}` + `${Second}`;
```

Comment:
notice the color change in your code?  We do not ever want to Capitalize our variables.  This actually sets up our vars to be something called CLASSES according to JS.  Also - can we solve this without `charAt()` ?